plugins/chat/app/services/chat_publisher.rb

```diff
@@ -12,7 +12,7 @@ def self.publish_new!(chat_channel, chat_message, staged_id)
         { scope: anonymous_guardian, root: :chat_message },
       ).as_json
     content[:type] = :sent
-    content[:stagedId] = staged_id
+    content[:staged_id] = staged_id
```

Comment:
generally trying to be very strict on casing difference between backend and frontend